paasta_tools/tron_tools.py

```diff
@@ -1169,15 +1169,48 @@ def validate_complete_config(
     return []
 
 
-def get_tron_namespaces(cluster, soa_dir):
+def _is_action_spark(action_info):
```

Comment:
nit: we should be typing new functions
```suggestion
def _is_action_spark(action_info: InstanceConfigDict) -> bool:
```